### PR TITLE
Data store

### PR DESCRIPTION
**Pull request by tinotenda6**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/data/Message.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package com.google.sps.data;
+
+/** A message received from contactme form */
+public final class Message {
+
+  private final long id;
+  private final String email;
+  private final String message;
+  ;
```
Comment (zhaonian) [inline portfolio/src/main/java/com/google/sps/data/Message.java]: remove extra `;`? :)

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -33,3 +33,16 @@ function showSlides() {
   slides[slideIndex-1].style.display = "block"; 
   setTimeout(showSlides, 2000); // Change image every 2 seconds
 }
+
+/** Check if email is valide */
+function validateEmail(email) {
```
Comment (zhaonian) [inline portfolio/src/main/webapp/script.js:38]: very cool!

File: walkthroughs/week-3-libraries/datastore/examples/todo-list/src/main/webapp/script.js [hunk 3]
```diff
@@ -17,6 +17,7 @@ function loadTasks() {
   fetch('/list-tasks').then(response => response.json()).then((tasks) => {
     const taskListElement = document.getElementById('task-list');
     tasks.forEach((task) => {
+        console.log(1);
```
Comment (zhaonian) [inline walkthroughs/week-3-libraries/datastore/examples/todo-list/src/main/webapp/script.js:20]: nit: indentation.

**Review by zhaonian — APPROVED** [review]

File: portfolio/src/main/java/com/google/sps/data/Message.java (hunk 1 above)
Comment (tinotenda6) [reply portfolio/src/main/java/com/google/sps/data/Message.java]: Thanks, I hadn't noticed it!

**Pull request merged**